.github/workflows/build-shared-android.yml

```diff
@@ -57,7 +61,8 @@ jobs:
       STRICT: ${{ inputs.STRICT }}
       ENV: ${{ inputs.ENV }}
       TERRASO_BACKEND: ${{ inputs.TERRASO_BACKEND }}
-      APP_VERSION: ${{ inputs.APP_VERSION }}
+      APP_BUILD: ${{ inputs.APP_BUILD }}
+      APP_VERSION: ${{ vars.APP_VERSION }}
```

Comment:
Questions: What's `vars`? When we want to make a new version number, where/how do we do that?

Reply:
In GitHub settings, we have both secrets and vars. To bump the version, we edit the settings there. (We can change this later if we decide to put the version in source.)